tests/wpt/meta/html/dom/idlharness.https.html.ini

```diff
@@ -1436,36 +1436,6 @@
   [SVGElement interface: attribute onbeforetoggle]
```

Comment:
Besides the interface, we should also check the tests at https://github.com/servo/servo/tree/650af7db926e9cc070e136f21a56bd47d566d10a/tests/wpt/tests/page-visibility

For this they need to be unskipped at https://github.com/servo/servo/blob/650af7db926e9cc070e136f21a56bd47d566d10a/tests/wpt/include.ini

Reply:
Here's my result:
```
Ran 11 tests finished in 11.0 seconds.
  • 5 ran as expected. 2 tests skipped.
  • 4 tests timed out unexpectedly
  • 6 tests had unexpected subtest results
```